keras_hub/src/layers/preprocessing/start_end_packer_test.py

```diff
@@ -40,7 +40,7 @@ def test_dense_2D_input(self):
             sequence_length=5, padding_side="left"
         )
         output = start_end_packer(input_data)
-        expected_output = [[0, 0, 5, 6, 7]]
+        expected_output = [[5, 6, 7, 0, 0]]
```

Comment:
Ditto, we are showing a lot of right hand side padding for the left padding option. I think we have introduced a bug.